test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -968,7 +968,7 @@ func waitUntilPodsScheduledInNamespace(ctx context.Context, podInformer coreinfo
 		if len(scheduled) >= wantCount {
 			return true, nil
 		}
-		klog.Infof("%s: namespace %s: got %d pods, want %d", name, namespace, len(scheduled), wantCount)
+		klog.Infof("%s: namespace %s, pods: want %d, got %d", name, namespace, wantCount, len(scheduled))
```

Comment:
no idea why this is using klog with the name of the tests instead of `b.Log`.

Anyways... it could be a separate refactor PR, if you have time for it.

Reply:
Sure, I'll follow this later.